src/utils/mod.rs

```diff
@@ -1,6 +1,7 @@
 //! Utilities used in this crate which can also be generally useful downstream.
 
 use alloc::string::String;
+pub use alloc::{format, vec};
```

Comment:
These macros were originally exported from the `string` and `collections` modules re-exported from `winter-utils`, so re-exporting them here isn't quite correct. See #292 for the patch that restores the original setup. I think @bobbinth is planning to merge both of those here shortly and cut new releases to fix downstream crates.

Reply:
oh great news then! I thought it they were intentionally removed from winterfell so this was more of a workaround. I'll close this PR then. Thanks for the explanation @bitwalker!

Reply:
btw, considering the deprecated modules will get removed eventually. Should we make a PR on miden-vm changing the usages of these re-exports for `alloc::***`, or do we just change the re-exports in miden-crypto?

Reply:
This is already done in https://github.com/0xPolygonMiden/miden-vm/pull/1277 - we just need to merge it an release it :)